src/Symfony/Component/Messenger/Command/FailedMessagesRemoveCommand.php

```diff
@@ -43,6 +43,10 @@ protected function configure(): void
     <info>php %command.full_name% {id1} [{id2} ...]</info>
 
 The specific ids can be found via the messenger:failed:show command.
+
+You can remove all messages from the failure transport by using the "--all" option (note that you also need the "--force" option):
+
+    <info>php %command.full_name% --all --force</info>
```

Comment:
can we do this differently: when `--force` is NOT passed, we ask something like: `Do you want to permanently remove the all (%d) messages?`
that would match the behavior when using ids and would allow removing this `--force` option from the help message

Reply:
I updated the code to match your idea. Currently, it displays the message count only if the receiver implements `MessageCountAwareInterface`. However, we may simplify this part by not displaying the message count when asking for confirmation. Suits me both ways